app/views/my/_work_action_menu.html.erb

```diff
@@ -28,13 +28,13 @@
         <span> TODO: Highlight Work on Profile </span>
       <% end %>
     </li>
-
     <li role="menuitem" tabindex="-1">
-      <%= link_to('#') do %>
-        <i class="glyphicon glyphicon-transfer" aria-hidden="true"></i>
-        <span> TODO: Transfer Ownership of Work </span>
+      <% gw = document.to_model %>
```

Comment:
@grosscol Do we need the to_model here?  That requires the work being loaded from fedora.